alpha/declcfg/write.go

```diff
@@ -18,6 +18,7 @@ import (
 type MermaidWriter struct {
 	MinEdgeName          string
 	SpecifiedPackageName string
+	CrossChannelLinking  bool
```

Comment:
I forgot that I left the cross-channel linking support in here.  It's not getting used by anything, but it ... also doesn't work. 
I'm open to leaving it as-is or breaking it out.

Reply:
Removing these entirely.  I had hoped that we'd be able to use this to visually debug the fact that we now can have `skips` in a channel which are not part of the channel (but within the Y range of the channel). 
But alas, it was just a bit too much effort for the time available.